### PR TITLE
fix tflite inference

### PR DESCRIPTION
**Pull request by fsx950223**

### CODE REVIEW

File: efficientdet/keras/inspector.py [hunk 1]
```diff
@@ -99,15 +99,24 @@ def main(_):
...
       if FLAGS.saved_model_dir.endswith('.tflite'):
+        is_tflite = True
         image_size = utils.parse_image_size(model_config.image_size)
         image_arrays = tf.image.resize_with_pad(image_arrays, *image_size)
         image_arrays = tf.cast(image_arrays, tf.uint8)
+    image_arrays = tf.expand_dims(image_arrays, axis=0)
     detections_bs = driver.serve(image_arrays)
     boxes, scores, classes, _ = tf.nest.map_structure(np.array, detections_bs)
+    if is_tflite:
+      classes, scores = scores, classes
```
Comment (fsx950223) [inline efficientdet/keras/inspector.py:114]: tflite nms outputs ```boxes, classes, scores, _ ```.
tf nms outputs ```boxes, scores, classes, _ ```.
Should it be fixed?

File: efficientdet/keras/train_lib.py [hunk 2]
```diff
@@ -33,120 +32,6 @@
 from keras import label_util
 from keras import postprocess
 from keras import util_keras
-from tensorflow_model_optimization.python.core.sparsity.keras import pruning_wrapper
```
Comment (fsx950223) [inline efficientdet/keras/train_lib.py:36]: I suspend tfmot research, so I clean up them.

**Comment by fsx950223** [comment]
I have done most parts of keras turtorial, maybe you should check it and fix some bugs.

**Comment by fsx950223** [comment]
Test link https://colab.research.google.com/drive/1yrrFJWnVX7GlufdeG-lCNKOs916kLgsA?usp=sharing

File: efficientdet/keras/efficientdet_keras_test.py [hunk 3]
```diff
@@ -192,6 +207,22 @@ def test_resample_feature_map(self):
             actual_result = resample_layer(feat, training, all_feats)
             self.assertAllCloseAccordingToType(expect_result, actual_result)
 
+  def test_hub_model(self):
+    image = tf.ones((1, 320, 320, 3))
```
Comment (fsx950223) [inline efficientdet/keras/efficientdet_keras_test.py]: The current branch has same output and gradients as original one

File: efficientdet/keras/efficientdet_keras_test.py [hunk 4]
```diff
@@ -192,6 +207,23 @@ def test_resample_feature_map(self):
             actual_result = resample_layer(feat, training, all_feats)
             self.assertAllCloseAccordingToType(expect_result, actual_result)
 
+  def test_hub_model(self):
+    image = tf.random.uniform((1, 320, 320, 3))
```
Comment (fsx950223) [inline efficientdet/keras/efficientdet_keras_test.py:211]: The current branch has same output and gradients as original one

**Comment by fsx950223** [comment]
I could train SOTA models in the branch, I will merge it. @mingxingtan
